crates/fj-viewer/Cargo.toml

```diff
@@ -28,6 +28,10 @@ path = "../fj-interop"
 version = "0.7.0"
 path = "../fj-math"
 
+[dependencies.fj-operations]
+version = "0.7.0"
+path = "../fj-operations"
+
```

Comment:
I think it's a bit unfortunate to add the dependency on `fj-operations` and (indirectly) `fj-kernel`. It would probably be better to move `ProcessedShape` to `fj-interop` instead.

Reply:
Yeah I didn't really like adding the dependency either. Let me give your suggestion a go.